job-service/client/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>job-service</artifactId>
+        <groupId>org.verapdf</groupId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>job-service-client</artifactId>
+
+
+</project>
```

Comment:
Let's make sure all our files ends with an empty line. You can configure IDEA to do that for you:
![image](https://user-images.githubusercontent.com/5430498/78266022-e37f0b00-750d-11ea-808c-7c73c609344d.png)